tools/depends/target/dav1d/Makefile

```diff
@@ -0,0 +1,52 @@
+include ../../Makefile.include
+include DAV1D-VERSION
+DEPS = ../../Makefile.include Makefile
+
+# configuration settings
+CONFIGURE = $(NATIVEPREFIX)/bin/python3 $(NATIVEPREFIX)/bin/meson \
+	--buildtype=release \
+	--prefix=$(PREFIX) \
+	--libdir=lib \
```

Comment:
i dont believe this is required. It will just automatically set to prefix/lib by default

Reply:
libdir is `lib64` on fedora

Reply:
Is this makefile used on Linux platforms though? I thought they used the cmakelists, and therefore the finddav1d module to add if enable_internal_dav1d is triggered?

Reply:
It can be used on linux platforms yes. Jenkins linux builds use the unified depends system so this is valid in my opinion (moved to the cross file though).

Reply:
All good then